src/docker_machine/src/docker_machine/opennebula.go

```diff
@@ -676,22 +678,25 @@ func (d *Driver) GetState() (state.State, error) {
 }
 
 func (d *Driver) Start() error {
-	controller := d.getController()
-
 	var err error
-	var vm_id int
 
-	if d.MachineId == 0 {
-		vm_id, err = controller.VMs().ByName(d.MachineName)
-		if err != nil {
-			return err
-		}
-		d.MachineId = vm_id
-	}
+	if drivers.MachineInState(d, state.Stopped)() {
```

Comment:
Probably this should take care of resuming also VMs in `Saved` state (i.e `STOPPED` and `SUSPENDED`). What do you think?

Reply:
Hi, I didn't implement other states, because the docker-machine has only methods start, stop, and restart. So VM should not be in the Saved state.

Reply:
Ok, it makes sense, thanks!